docs-src/content/using-cubic-sdk/streaming.md

```diff
@@ -111,6 +111,73 @@ for resp in client.StreamingRecognize(cfg, audio):
             print("\r{0}".format(result.alternatives[0].transcript), end="\n")
 {{< /tab >}}
 
+{{< tab "C++" "c++" >}}
+#include "cubic_client.h"
+
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <thread>
+
+namespace CubicPB = cobaltspeech::cubic;
+const std::string serverAddress = "localhost:2727";
+const std::string filename = "test.raw";
+
+// This client demonstrates using synchronous recognition.
+int main(int argc, char *argv[]) {
+    // Create the client (note this is an insecure connection,
+    // which is not recommended for production).
```

Comment:
A lot of these comments tend to end _without_ a period, but some of them do end in a period.  Do we want to be consistent on that?

Reply:
I don't have strong feelings on it. I can add the periods to be consistent and grammatical.